packages/core/src/components/GlobalActions/GlobalActions.stories.tsx

```diff
@@ -183,3 +201,113 @@ export const SectionVariant: StoryObj<HvGlobalActionsProps> = {
     );
   },
 };
+
+export const Breadcrumb: StoryObj<HvGlobalActionsProps> = {
+  argTypes: {
+    classes: { control: { disable: true } },
+  },
+  parameters: {
+    docs: {
+      description: {
+        story:
+          "The `title` property can be a custom component. For example, here we use a `HvBreadCrumb` component.",
```

Comment:
Not a component, element/ReactNode
```suggestion
          "The `title` property can be a custom element. For example, here we use a `HvBreadCrumb` component.",
```